src/sentry/event_manager.py

```diff
@@ -2033,10 +2041,14 @@ def _save_aggregate_performance(jobs: Sequence[Performance_Job], projects):
                         metric_tags["create_group_transaction.outcome"] = "no_group"
 
                         problem = performance_problems_by_fingerprint[new_grouphash]
-                        kwargs["type"] = problem.type.value
-                        kwargs["data"]["metadata"]["title"] = f"N+1 Query: {problem.desc}"
+                        group_kwargs = kwargs
```

Comment:
We still need to copy this first, `kwargs.copy()`

Reply:
oh excuse my JS habits 😶‍🌫️